generators/app/templates/sources/main/_main.run.ts

```diff
@@ -84,6 +87,21 @@ function main($window: ng.IWindowService,
     updateTitle($state.current.data ? $state.current.data.title : null);
   });
 
+  /**
+   * Enables tracking by analytics service.
+   */
+  // HACK : ignore the first $viewContentLoaded event because it's actually fired once when uiView is instantiated,
+  // and then it's fired a second time after is has been linked. This is "by design" :-/
+  // (http://stackoverflow.com/questions/31000417/angular-js-viewcontentloaded-loading-twice-on-initial-homepage-load)
+  let loadedOnce = false;
+  vm.$on('$viewContentLoaded', function () {
```

Comment:
As per ui-router documentation ([link](https://github.com/angular-ui/ui-router/wiki)) :
> State change events are deprecated, DISABLED and replaced by Transition Hooks as of version 1.0
So maybe the real solution is to look into transition hooks ? But that's a different beast altogether.